Makefile

```diff
@@ -36,78 +44,79 @@ fmt: goimports installcue
 	$(CUE) fmt ./pkg/stdlib/pkgs/*
 	$(CUE) fmt ./pkg/stdlib/op.cue
 	$(CUE) fmt ./pkg/workflow/tasks/template/static/*
-# Run go vet against code
 
+## sdk_fmt: Run go fmt against code
 sdk_fmt:
 	./hack/sdk/reviewable.sh
 
+## vet: Run go vet against code
 vet:
 	@$(INFO) go vet
 	@go vet $(shell go list ./...|grep -v scaffold)
 
+## staticcheck: Run the staticcheck
 staticcheck: staticchecktool
 	@$(INFO) staticcheck
 	@$(STATICCHECK) $(shell go list ./...|grep -v scaffold)
 
+## lint: Run the golangci-lint
 lint: golangci
 	@$(INFO) lint
 	@$(GOLANGCILINT) run --skip-dirs 'scaffold'
 
+## reviewable: Run the reviewable
 reviewable: manifests fmt vet lint staticcheck helm-doc-gen sdk_fmt
 	go mod tidy
 
-# Execute auto-gen code commands and ensure branch is clean.
+# check-diff: Execute auto-gen code commands and ensure branch is clean.
 check-diff: reviewable
 	git --no-pager diff
 	git diff --quiet || ($(ERR) please run 'make reviewable' to include all changes && false)
 	@$(OK) branch is clean
 
-# Push the docker image
+## docker-push: Push the docker image
 docker-push:
-	docker push $(VELA_CORE_IMAGE)
-
-
+	@echo "===========> Pushing docker image"
+	@docker push $(VELA_CORE_IMAGE)
 
+## image-cleanup: Delete Docker images
 image-cleanup:
-ifneq (, $(shell which docker))
-# Delete Docker images
-
-ifneq ($(shell docker images -q $(VELA_CORE_TEST_IMAGE)),)
-	docker rmi -f $(VELA_CORE_TEST_IMAGE)
-endif
-
-ifneq ($(shell docker images -q $(VELA_RUNTIME_ROLLOUT_TEST_IMAGE)),)
-	docker rmi -f $(VELA_RUNTIME_ROLLOUT_TEST_IMAGE)
-endif
-
-endif
-
+	ifneq (, $(shell which docker))
+		ifneq ($(shell docker images -q $(VELA_CORE_TEST_IMAGE)),)
+			@docker rmi -f $(VELA_CORE_TEST_IMAGE)
+		endif
 
+		ifneq ($(shell docker images -q $(VELA_RUNTIME_ROLLOUT_TEST_IMAGE)),)
+			@docker rmi -f $(VELA_RUNTIME_ROLLOUT_TEST_IMAGE)
+		endif
+	endif
```

Comment:
You break the makefile grammar here. `ifneq` and such primitives should have no spaces ahead. This breaks the tests.